### ISSUE TITLE
Credo should not complain about ParenthesesInCondition in one-line `if` call

### ISSUE THREAD

**Issue by rrrene** [rrrene/credo#225]
While it is correct that this triggers a Credo issue:

```elixir
      if( valid?(username) ) do
        # ...
      end
```

this should not trigger and issue:

```elixir
props =
  if(valid?(username), do: [:authorized]) ++
  if(admin?(username), do: [:admin])
```

This was originally reported by @josevalim in [this comment](https://github.com/elixir-lang/ex_doc/pull/639#discussion_r87639310).

@sstern6 I added a (now failing) test reproducing this in 12ef530c14a57504603652efbadc9fc88d75a165. I dabbled a bit to find a quick fix, but did not come up with anything satisfying. Feel free to give it a try :+1:

**Comment by sstern6** [comment]
Thanks @rrrene a little short on time at the moment and have a lot to catch up on for credo but will put it on my list of things to do!  If anyone wants to take this on though feel free!
